gqlalchemy/disk_storage.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2016-2021 Memgraph Ltd. [https://memgraph.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sqlite3
+import contextlib
+
+from typing import Optional, List
+
+
+class OnDiskPropertyDatabase:
+    def save_node_property(self, node_id: int, property_name: str, property_value: str):
+        pass
+
+    def load_node_property(self, node_id: int, property_name: str, property_value: str):
+        pass
+
+    def save_relationship_property(self, relationship_id: int, property_name: str, property_value: str):
+        pass
+
+    def load_relationship_property(self, relationship_id: int, property_name: str, property_value: str):
+        pass
+
+    def drop_database(self):
+        pass
+
+
+class SQLitePropertyDatabase(OnDiskPropertyDatabase):
+    def __init__(self, database_name: str = "on_disk_properties.db"):
+        self.database_name = database_name
+        self.create_node_property_table()
+        self.create_relationship_property_table()
+
+    def execute_query(self, query) -> List[str]:
+        with contextlib.closing(sqlite3.connect(self.database_name)) as conn:
+            with conn:  # autocommit changes
+                with contextlib.closing(conn.cursor()) as cursor:
+                    cursor.execute(query)
+                    return cursor.fetchall()
+
+    def create_node_property_table(self):
```

Comment:
if nothing is returned I suggest annotating it
```suggestion
    def create_node_property_table(self) -> None:
```